vnext/Microsoft.ReactNative/Modules/CreateModules.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <CreateModules.h>
+#include <Modules/WebSocketModule.h>
+#include <Modules/WebSocketModuleUwp.h>
+#include <WinRTWebSocketResource.h>
+
+// React Native
+#include <cxxreact/CxxModule.h>
+
+// Windows API
+#include <winrt/Windows.Security.Cryptography.Certificates.h>
+
+namespace Microsoft::React {
```

Comment:
I think new code in Microsoft.ReactNative uses the Microsoft::ReactNative namespace. There's still old stuff using ReactUwp though.